content/post/37-ipfs-conf-dev-meetings-and-user-registry.md

```diff
@@ -5,11 +5,11 @@ title: "Announcements: IPFS Conf,  Developer Meetings for IPFS & libp2p and a Us
 author: flyingzumwalt
 ---
 
-We’re very excited to announce two events that have been in the making for a long time. In (tentatively) November we will host a public **IPFS Conf** in Lisbon. To prepare for that event, in July we will convene small **IPFS and libp2p Developer Meetings** in Berlin. All of these events will be loosely inspired by PyCon and Docker Conf — events where community members play a very active role in shaping the agenda, and where a large portion of time is allocated to working meetings where Working Groups and product teams make the most of being together in-person.
+We’re very excited to announce two events that have been in the making for a long time. In ~~(tentatively) November~~ 2019 we will host a public **IPFS Conf** in Lisbon. To prepare for that event, in July we will convene small **IPFS and libp2p Developer Meetings** in Berlin. All of these events will be loosely inspired by PyCon and Docker Conf — events where community members play a very active role in shaping the agenda, and where a large portion of time is allocated to working meetings where Working Groups and product teams make the most of being together in-person.
```

Comment:
We shouldn't commit to November, we're likely to do it earlier than that.

Reply:
My change is to strike through the ~~(tentatively) November~~ that was present before and add the 2019 not mentioning a month.